### PR TITLE
Python 3 compatibility for vellum-to-hq

### PR DESCRIPTION
**Pull request by millerdev**
@dimagi/py3 cc @sravfeyn

### CODE REVIEW

**Comment by millerdev** [comment]
Killed the build. There are not tests for `vellum-to-hq`

File: scripts/vellum-to-hq [hunk 1]
```diff
@@ -226,5 +232,12 @@ def get_version(vellum_tar):
         return ""
 
 
+def byte_stream(stream):
+    # In Python 3 get a stream that accepts bytes (`sys.stdout.buffer`).
+    # In Python 2 output streams do not have a "buffer" attribute.
+    # https://github.com/amoffat/sh/issues/242
+    return getattr(stream, "buffer", stream)
```
Comment (nickpell) [inline scripts/vellum-to-hq]: I generally prefer to use ```if six.PY2``` for logic that branches on python version in order to make it clear what code can be deleted post-upgrade.

**Review by nickpell — APPROVED** [review]

File: scripts/vellum-to-hq (hunk 1 above)
Comment (millerdev) [reply scripts/vellum-to-hq]: d70df42085db17de7420a77b773b2f01f49d0093

**Review by nickpell — APPROVED** [review]
Thanks!

**Pull request merged**